content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: An Alternative Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Admin
+description: A new deployment model for Istio.
+publishdate: 2020-08-26
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,install,deploy,'1.7']
+---
+
+## Overview
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
```

Comment:
Comment disappeared on commit change and I don't seem to be able to add more text there.

I still think you would benefit from using "Service owner" so they don't all start with "Mesh" (b) I'm not sure I've seen evidence that Mesh Operator and Mesh Admin are two personas, _unless_ they're already consuming a managed service, or something like this.

Reply:
I think in this case, they are because one team owns control plane install/upgrade, one team managed the mesh data plane.